how-to-release.md

```diff
@@ -0,0 +1,298 @@
+* Once the operation completes, locate the URL field in the Summary tab of the newly closed repository (it will be something like https://oss.sonatype.org/content/repositories/iopravega-XXXX where XXXX is the number of the staging repository). This should be tested to ensure that all artifacts are present and functions as expected.
+
+## Staging the docker images
+
+The docker images should also be staged to the release manager's personal repository for validation from Docker/Kubernetes users. This can be done with the below command.
+```
+./gradlew clean docker dockerPush -PpravegaBaseTag=<docker-username>/pravega -PbookkeeperBaseTag=<docker-username>/bookkeeper -PpravegaVersion='X.Y.Z-rcN'
+```
+
+# Vote on the release candidate
+
+## Announcing for voting
+Once you have built and individually reviewed the release candidate, please share it for the community-wide review. You need to share these things to the dev mailing list for this release candidate and reference the link in the Slack channel.
+- The link of the github release candidate tag
+- The Sonatype staging repository
+- Docker image repository
+- The release notes
+
+Here is the Email template.
+```
+Hi everyone,
+Please review and vote on the release candidate #0 for the version 0.3.0, as follows:
++1, Approve the release
+-1, Do not approve the release (please provide specific comments)
+
+Github link: https://github.com/pravega/pravega/releases/tag/v0.3.0-rc0
+Sonatype link: https://oss.sonatype.org/content/repositories/iopravega-1004
+Docker Image: brianzhou/pravega:0.3.0-rc0
+Release notes: https://gist.github.com/crazyzhou/f53ccfc395ce35f4d09bc2a27dsbcf69
+```
+
+## Voting process
+The community including the release manager will be responsible for verifying the release candidate. Anyone can vote +1 or -1 on the release candidates, which helps to gain more confidence on the release. The vote happens on the dev mailing list. The necessary conditions for the release vote to pass are shown [here](https://github.com/pravega/.github/blob/main/governance.md#actions).
+
+There are a few things to check including
+
+- No open release-blocking issues
+- All the unit and integration tests pass
+- Checksums of the artifacts and source code are correct
+- Verify that it can locate, build and run with the staging artifacts. For example, use the [Pravega samples](https://github.com/pravega/pravega-samples) to verify.
+- Docker images can run in a container successfully
+
+It is possible that a release candidate is problematic or denied. In this case, you need to announce the cancellation of this vote, and then fixes issues and repeats the cycle.
+
+Once the community passes the vote, you need to announce the vote passing in both the mailing list and Slack channel.
```

Comment:
Once the community passes the vote, you announce the vote result in both the dev mailing list and in the release Slack channel.